frontend/src/components/LoginAndRegistrationPage/LoginPage/LoginForm.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
-import { ErrorMessage, Formik, Field } from 'formik';
+import { Formik } from 'formik';
 import { Container, Form, Row, Col, Button, Spinner } from 'react-bootstrap';
+import { FormCol } from '../FormCol';
 
 export default function LoginForm() {
     return (
         <Formik
             initialValues={{
-                email: '',
+                opinion: '',
```

Comment:
Dlaczego dodałeś "opinion"? Nie widzę żeby to było gdzieś używane, chyba że nie dostrzegam to mnie popraw 😅

Reply:
Literówka, musiałem pomylić pliki